packages/formik/src/ErrorMessage.tsx

```diff
@@ -4,6 +4,7 @@ import { getIn, isFunction } from './utils';
 import { connect } from './connect';
 
 export interface ErrorMessageProps {
+  id?: string;
```

Comment:
Should we just allow all typical `div` attributes and pull off our specific ones during render?

Reply:
@jaredpalmer

Reply:
```ts
export interface ErrorMessageProps extends React.HTMLAttributes<HTMLDivElement> {
  component?: string | React.ComponentType;
  children?: (errorMessage: string) => React.ReactNode;
  name: string;
  render?: (errorMessage: string) => React.ReactNode;
}
```

Not sure if there's a native-specific version of this?